### ISSUE TITLE
Show version mismatch errors in Argo again

### ISSUE THREAD

**Issue by andrewjbtw** [sul-dlss/argo#1762]
**Describe the bug**
Argo used to show version mismatch errors in the interface for items that have version mismatch errors at sdr-ingest-transfer in the accessionWF. 

These errors are now being reported as problems with a background job with no further detail:

![Screen Shot 2019-12-20 at 10 00 20 AM](https://user-images.githubusercontent.com/8975583/71280357-caac6a00-230f-11ea-9a86-72a7755eaff4.png)

**User Impact**
Without a clear error message, it's harder to figure out what needs to be done.

**To Reproduce**
I've tried to reproduce this in stage, but it's difficult to do it intentionally via Argo because that system has safeguards to prevent it. The most common cause of a version mismatch has been when an external system (like Goobi) triggers a new accessionWF without incrementing the version.

In the specific case screenshotted above, it's not clear what caused the error since the object was accessioned only once via preassembly.

**Expected behavior**
Version mismatch errors are reported in the Argo user interface.

**Additional context**
This issue might apply to all errors that happen in background jobs. Other errors for sdr-ingest-transfer, like "content dir not found" might also be reported only as "problem with background job".

**Comment by jcoyne** [comment]
The change needs to be made here: https://github.com/sul-dlss/dor-services-app/blob/master/app/jobs/log_failure_job.rb#L22-L23  Or, we need to alter argo to show `error_text` on the workflow process.

**Comment by peetucket** [comment]
What's an example druid (in either prod or stage)?

**Comment by andrewjbtw** [comment]
In prod: https://argo.stanford.edu/view/druid:jm128jc2929

**Comment by peetucket** [comment]
It appears the `error_text` column in the workflow table is not currently returned from API calls to `workflow-server-rails`.  So if we wanted to log in the way noted in https://github.com/sul-dlss/dor-services-app/blob/master/app/jobs/log_failure_job.rb#L22-L23 and have that `error_text` shown in Argo, we would need to:

1. Update worklow-server-rails to return that column in API calls
2. Update argo models to parse that column into the data structure used by the views
3. Update argo display to show that column

Alternatively, we could update https://github.com/sul-dlss/dor-services-app/blob/master/app/jobs/log_failure_job.rb#L22-L23 to simply stuff that rest of that output into the `error_msg`.  While not great, we already do big text error output in other places (e.g. the `preservation-audit` step of `preservationAuditWF` already barfs a giant error message to that column with all of the failures).  For example, from `workflow-server-rails`:

```
2.5.3 :010 > WorkflowStep.where(druid:'druid:fk847cz0044',workflow:'preservationAuditWF',version:7).first.error_msg
 => "validate_checksums (actual location: services-disk03; ) /services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/manifests/signatureCatalog.xml refers to file (/services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/data/content/00002407_0001.tif) not found in Moab && /services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/manifests/signatureCatalog.xml refers to file (/services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/data/content/00002407_0001.txt) not found in Moab && /services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/manifests/signatureCatalog.xml refers to file (/services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/data/content/00002407_0002.tif) not found in Moab && /services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/manifests/signatureCatalog.xml refers to file (/services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/data/content/00002407_0002.txt) not found in Moab && /services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/manifests/signatureCatalog.xml refers to file (/services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/data/content/00002407_0003.tif) not found in Moab && /services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/manifests/signatureCatalog.xml refers to file (/services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/data/content/00002407_0003.txt) not found in Moab && /services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/manifests/signatureCatalog.xml refers to file (/services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/data/content/00002407_0004.tif) not found in Moab && /services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/manifests/signatureCatalog.xml refers to file (/services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/data/content/00002407_0004.txt) not found in Moab && /services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/manifests/signatureCatalog.xml refers to file (/services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/data/content/00002407_0005.tif) not found in Moab && /services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/manifests/signatureCatalog.xml refers to file (/services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/data/content/00002407_0005.txt) not found in Moab && /services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/manifests/signatureCatalog.xml refers to file (/services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/data/content/00002407_0006.tif) not found in Moab && /services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/manifests/signatureCatalog.xml refers to file (/services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/data/content/00002407_0006.txt) not found in Moab && /services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/manifests/signatureCatalog.xml refers to file (/services-disk03/sdr2objects/fk/847/cz/0044/fk847cz0044/v0001/data/content/00002407_0007.tif) not found in Moab && /services-disk03/sdr2objects/fk/847/cz/0044 [etc] [etc] .......
```

**Comment by justinlittman** [comment]
`error_text` is a binary field and according to @jcoyne it's intended use is unknown. Given that, I'm going to stop using it in dor-services-app and merge into a single error message.

**Comment by justinlittman** [comment]
See https://github.com/sul-dlss/dor-services-app/pull/578

**Comment by peetucket** [comment]
Need to confirm if this is fixed by the PR and deployment
